src/server/test/compaction_filter_rule_test.cpp

```diff
@@ -131,5 +131,43 @@ TEST(ttl_range_rule_test, match)
         ASSERT_EQ(rule.match("", "", svalue.parts[0]), test.match);
     }
 }
+
+TEST(compaction_filter_rule_test, create)
+{
+    const uint32_t data_version = 1;
+
+    // ttl_range_rule
+    std::string params_json = "{\"start_ttl\":1000,\"stop_ttl\":2000}";
```

Comment:
I recommend using [string_literal](https://en.cppreference.com/w/cpp/language/string_literal) to simplify the json string.

Reply:
Done